README.md

```diff
@@ -788,14 +789,21 @@ scala> val low = 40.dollars / megawattHour
 low: squants.market.Price[squants.energy.Energy] = 40.00 USD/1.0 MWh
 ```
 
-Implicit conversion support for using Double on the left side of multiplication:
+Implicit conversion support for using Doubles, Longs and BigDecimals on the left side of multiply and divide operations:
```

Comment:
One thing I noticed is 

`BigDecimal(1) / hour` works in the tut, but
`BigDecimal(1) per hour` does not:  method not defined.

However, both of these are defined in the same implicit class and both work in the console.  For the README, I just went back to using the / operator.  Any thoughts on that?